src/sentry/static/sentry/app/views/settings/project/navigationConfiguration.jsx

```diff
@@ -116,4 +115,13 @@ export default function getConfiguration({project}) {
       ],
     },
   ];
+
+  // Remove saved-searches if the org is on org-level saved searches
+  if (organization.features && organization.features.includes('org-saved-searches')) {
```

Comment:
In the object declaration for saved searches, you can do `show: ({organization}) => organization.features.includes('org-saved-searches')`